wallet/src/wallet-api/routes.rs

```diff
@@ -744,9 +744,16 @@ async fn newasset(
 }
 
 async fn buildsponsor(
+    options: &NodeOpt,
     bindings: &HashMap<String, RouteBinding>,
     wallet: &mut Option<Wallet>,
 ) -> Result<(sol::AssetDefinition, String), tide::Error> {
+    // Make sure the EQS is running.
+    surf::connect(options.eqs_url())
+        .send()
+        .await
+        .expect("Cannot build the sponsor transaction since EQS is down");
```

Comment:
Ah, this is why the tests need the EQS to be running. Seems like this is another thing that should be abstracted by `CapeWalletBackend`. It doesn't make much sense for the tests to ping the real EQS when the mock backend is talking to a mock query service. Not a big deal, though, since it only amounts to a slight inefficiency in test code.

Reply:
Yeah, it seems better to ping the EQS by `CapeWalletBackend` rather than `buildsponsor`. I'll modify this.